src/Glpi/Form/Condition/ConditionHandler/ItemConditionHandler.php

```diff
@@ -93,11 +93,12 @@ public function applyValueOperator(
     #[Override]
     public function convertConditionValue(string $value): int
     {
+        $nameFields = [];
         $item = getItemForItemtype($this->itemtype);
-        $nameFields = [$item->getNameField()];
         if ($item->isField($item->getCompleteNameField())) {
             $nameFields[] = $item->getCompleteNameField();
         }
+        $nameFields[] = $item->getNameField();
```

Comment:
If `$item->getCompleteNameField();` and `$item->getNameField()` have the same value (I don't know if it's possible - seems it is), should not we prevent duplication?